robottelo/vm.py

```diff
@@ -16,7 +16,7 @@
 from robottelo.common import conf, ssh
 
 
-BASE_IMAGES = ('rhel7', 'rhel65', 'rhel66')
+BASE_IMAGES = ('rhel71', 'rhel70', 'rhel66', 'rhel65')
```

Comment:
I think Sat only currently support RHEL 7.1 and RHEL 6.6. Maybe @elyezer knows better?